stable/aws-for-fluent-bit/README.md

```diff
@@ -45,19 +45,25 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `filter.*` | Values for kubernetes filter | |
 | `filter.extraFilters` | Append to existing filter with value |
 | `additionalFilters` | Adding more filters with value |
-| `cloudWatch.enabled` | Whether this plugin should be enabled or not [details](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit) | `true` | ✔
+| `cloudWatch.enabled` | Whether this plugin should be enabled or not [details](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch) | `true` | ✔
```

Comment:
Since this links back to fluent-bit and to maintain naming convention consistency I would suggest to use `cloudWatchLogs.enabled` for newer plugin which aligns with this repos naming convention and also with fluent-bit naming convention

Reply:
we can `cloudWatch.enabled` which points to old plugin while `cloudWatchLogs.enabled` which points to new plugin. This way it aligns to fluentbit as well.